org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServiceAccessor.java

```diff
@@ -421,18 +421,18 @@ private static interface ServerSupplier {
 	}
 
 	/**
-	 * Returns {@code true} if there are running language servers satisfying a capability predicate. This does not
-	 * start any matching language servers.
+	 * Returns {@code true} if there are running language servers satisfying a capability predicate.
+	 * This does not start any matching language servers.
 	 *
 	 * @param request
 	 * @return {@code true} if there are running language servers satisfying a capability predicate
 	 */
 	public static boolean hasActiveLanguageServers(Predicate<ServerCapabilities> request) {
-		return !getLanguageServers(null, request, true).isEmpty();
+		return !getLanguageServers(null, request).isEmpty();
 	}
 
 	public static boolean hasActiveLanguageServers(IFile file, Predicate<ServerCapabilities> request) {
-		return !getLanguageServers(null, request, true).isEmpty();
+		return !getLanguageServers(file.getProject(), request).isEmpty();
```

Comment:
This makes so, but we could call so far with a `null` file. See one of the test failures:

java.lang.NullPointerException: Cannot invoke "org.eclipse.core.resources.IFile.getProject()" because "file" is null
	at org.eclipse.lsp4e.LanguageServiceAccessor.hasActiveLanguageServers(LanguageServiceAccessor.java:435)

Could you add a @Nullable annotation and handle the case?